regression-tests.recursor-dnssec/test_ExtendedErrors.py

```diff
@@ -119,6 +119,19 @@ def testExpired(self):
             self.assertEqual(res.options[0].otype, 15)
             self.assertEqual(res.options[0], extendederrors.ExtendedErrorOption(7, b''))
 
+    def testAllExpired(self):
+        qname = 'servfail.nl.'
```

Comment:
we really shouldn't rely on external domains in our tests

Reply:
I think some of the existign tests already do or didn't I look close enough?

Reply:
Not that I do not agree, but setting up something for these tests not relying on external servers is quite some work.

Reply:
#10983